.github/workflows/release-rancher-desktop.yml

```diff
@@ -0,0 +1,47 @@
+name: Build and Release Rancher Dashboard - Rancher Desktop Embed
+
+on:
+  push:
+    tags:
+        - 'v**-desktop*'
```

Comment:
Going forward we may use the approach for other products so it may make sense to group them by product name first instead - `desktop-v*`. This would make it easier to dig through releases in the github ui and it would avoid lots of tags with non-dashboard versions first

Reply:
Sounds like a good approach to me - fixed